src/components/AddForm/AddForm.jsx

```diff
@@ -21,6 +26,22 @@ function AddForm({
     const [inputValue, setInputValue] = useState('');
     const [deadline, setDeadline] = useState('');
     const [validMsg, setValidMsg] = useState('');
+    const [dangerClass, setDangerClass] = useState(false);
+
+    useEffect(() => {
+        switch (validMsg) {
+
+            case 'Empty tasks Date!':
+                setDangerClass(true);
+                break;
+            case 'Greeting time Travel!':
+                setDangerClass(true);
+                break;
+            case 'Empty tasks Name!':
+                setDangerClass(true);
```

Comment:
1. Why do you need all this cases? It does same thing in all cases
2. Try to avoid switch statement. It's ok, it does what it does but old fashioned style and make your code bigger and hard to enhance

Reply:
example:
```
const something = {
   empty: 'Can\'t be empty',
   short: 'Too short',
   specialChars: 'Can not contain special characters',
   ...
}

something[value]
```

Reply:
try to avoid texts for switch cases and object keys.
Good practice is to use constants or literals like this or example above:

```
const EMPTY = 'empty';
const SHOR = 'short';
...

{
    EMPTY: 'Can\'t be empty',
    SHORT: 'Too short',
    SPECIAL_CHARS: 'Can not contain special characters',
}
```